aws/data_source_aws_lambda_alias.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
Since this was submitted, there has been a few changes to Terraform AWS Provider development, including the import path for the Terraform Plugin SDK. This will be handled on merge along with a few other modernization changes that are required (NodeJS 8 deprecation in testing, etc).